directory/src/main/java/org/ecclesia/directory/domain/Location.java

```diff
@@ -1,8 +1,10 @@
 package org.ecclesia.directory.domain;
 
-import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
-@Data
+@Getter @Setter @NoArgsConstructor
```

Comment:
**Reporter**: Spotbugs
**Rule**: SE_BAD_FIELD
**Severity**: WARN
**File**: org/ecclesia/directory/domain/Location.java L7
**Source**: org.ecclesia.directory.domain.Location

Non-transient non-serializable instance field in serializable class&#10;&#10;&#10;&lt;p&gt; This Serializable class defines a non-primitive instance field which is neither transient,&#10;Serializable, or &lt;code&gt;java.lang.Object&lt;/code&gt;, and does not appear to implement&#10;the &lt;code&gt;Externalizable&lt;/code&gt; interface or the&#10;&lt;code&gt;readObject()&lt;/code&gt; and &lt;code&gt;writeObject()&lt;/code&gt; methods.&amp;nbsp;&#10;Objects of this class will not be deserialized correctly if a non-Serializable&#10;object is stored in this field.&lt;/p&gt;&#10;


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1577714371>*